src/StackAdmin/Azs.Storage.Admin/Module/Azs.Storage.Admin/Azs.Storage.Admin.psd1

```diff
@@ -6,142 +6,130 @@ Licensed under the MIT License. See License.txt in the project root for license
 
-    # Description of the functionality provided by this module
-    Description       = 'Storage Admin Client'
+# Description of the functionality provided by this module
+Description = 'Storage Admin Client'
 
-    # Minimum version of the Windows PowerShell engine required by this module
-    PowerShellVersion = '5.0'
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '5.0'
 
-    # Name of the Windows PowerShell host required by this module
-    # PowerShellHostName = ''
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
 
-    # Minimum version of the Windows PowerShell host required by this module
-    # PowerShellHostVersion = ''
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
 
-    # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-    # DotNetFrameworkVersion = ''
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# DotNetFrameworkVersion = ''
 
-    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-    # CLRVersion = ''
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
 
-    # Processor architecture (None, X86, Amd64) required by this module
-    # ProcessorArchitecture = ''
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
 
-    # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.8.3'; },
-        @{ModuleName = 'AzureRM.Resources'; RequiredVersion = '6.4.3'; })
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.8.3'; },
+	@{ModuleName = 'AzureRM.Resources'; RequiredVersion = '6.4.3'; })
 
-    # Assemblies that must be loaded prior to importing this module
-    RequiredAssemblies = @('.\ref\fullclr\Microsoft.AzureStack.Management.Storage.Admin.dll')
+# Assemblies that must be loaded prior to importing this module
+RequiredAssemblies = @('.\ref\fullclr\Microsoft.AzureStack.Management.Storage.Admin.dll')
 
-    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
-    # ScriptsToProcess = @()
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
 
-    # Type files (.ps1xml) to be loaded when importing this module
-    TypesToProcess    = @('Generated.PowerShell.Commands\FormatFiles\Azs.Storage.Admin.Type.ps1xml')
```

Comment:
Dont you need the custome formatting anymore?

Reply:
The original customer formatting is used for metrics, but we remove metrics from srp to mdm in the new version, so I manually remove this.